### PR TITLE
feat: onboarding

### PR DESCRIPTION
**Pull request by philipdesmedt**
This is work in progress.

@Macxim can continue on this branch

### CODE REVIEW

File: scripts/utils.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ async function getNonce(address) {
 
 function resolveUrl() {
   if (env === 'mocknet') {
-    return 'http://localhost:20080';
+    return 'http://localhost:3999';
```
Comment (victorcoder) [inline scripts/utils.js:41]: back to `3999`?

Comment (philipdesmedt) [reply scripts/utils.js:41]: "Unfortunately" yes. The clarinet release went back to 3999 which is the default. Perhaps we can configure it through an ENV variable so every contributor can choose what to do

**Review by victorcoder — APPROVED** [review]
LGTM

**Comment by nieldeckx** [comment]
Awesome 👍 

@Macxim Copy is important here. Suggest you work with Pieter on this so it can be finished completely.

**Comment by philipdesmedt** [comment]
@Macxim @nieldeckx this is ready for review imho

added a check in localStorage to redirect automatically if the user is logged in & hasn't done the onboarding yet.

**Comment by philipdesmedt** [comment]
@Macxim @nieldeckx actually don't merge yet since I found a bug when you are logging in and it doesn't redirect yet

**Comment by Macxim** [comment]
It's still missing a final message to _close_ the onboarding and invite the user to use the app.

**Comment by philipdesmedt** [comment]
@Macxim good point - can you add that?

**Comment by Macxim** [comment]
Will do in a bit.

**Comment by Macxim** [comment]
Added the end step of the onboarding. Feel free to adapt the copy if needed!

**Comment by philipdesmedt** [comment]
I fixed a bug where it wouldn't redirect you to the onboarding when logging in.

@Macxim last page looks great. The only thing is I have to click "Next" twice on the last governance page to get to the last page. Do you have this as well?

**Comment by Macxim** [comment]
Should be fixed now 🚀

**Comment by philipdesmedt** [comment]
@nieldeckx any last comments on this?

**Comment by Macxim** [comment]
I would like to fix the responsive 😄 But I guess it can be done in another PR.

**Comment by philipdesmedt** [comment]
@Macxim yeah - although responsiveness is not as high prio since the Hiro Web Wallet only works on FF, Chrome and Brave so far...

**Review by Macxim — APPROVED** [review]

**Comment by philipdesmedt** [comment]
Just ran through this again. LGTM. Gonna merge

**Pull request merged**